Scripts/build-archive.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
I did some improvements after the feature, you can pick up a new version from the SDK repo